src/apps/Calculators/AxByEqualsP/AxByModeReadoutTable.js

```diff
@@ -0,0 +1,67 @@
+import { handleSetPlayerClick } from "../../../eventHandlers";
+import generateAxByScl from "./util/generateAxByScl";
+
+const AxByModeReadoutTable = ({
+  index,
+  ltObject,
+  sclData,
+  stateData,
+  setStateData,
+}) => {
+  const setPlayerObject = {
+    tool: "axby",
+    scale: ltObject.mode,
+    sclData: sclData,
+    mode: index + 1,
+  };
+  return (
+    <div>
```

Comment:
Similarly - if this `<div>` is just here to avoid the `JSX expressions must have one parent element` issue then you can use a [fragment](https://reactjs.org/docs/fragments.html) instead:

![Uploading image.png…]()